spec/install/force_spec.rb

```diff
@@ -2,19 +2,41 @@
 
 describe "bundle install" do
   describe "with --force" do
-    before :each do
+    it "re-installs installed gems" do
       gemfile <<-G
         source "file://#{gem_repo1}"
         gem "rack"
       G
+      bundle "install"
 
+      bundle "install --force"
+      expect(out).to include "Installing rack 1.0.0"
+      should_be_installed "rack 1.0.0"
+    end
+
+    # How do I test this
+    it "deletes installed gems" do
+      gemfile <<-G
+        source "file://#{gem_repo1}"
+        gem "rack"
+      G
       bundle "install"
+
```

Comment:
On this line, add a file into the directory where `rack` is installed. IIRC, that's something like `default_bundle_path("gems/rack-1.0.0/garbage").open{|f| f.write("blah blah blah") }`.